bt_lession1/Ex3/ex3.js

```diff
@@ -1,7 +1,7 @@
 /** @format */
 
 // + Data 1: Dolphins có điểm là 96, 108 và89. Koalas có điểm là  88, 91 và 110
-// Tính ĐTB cả 2 đội
+// Tính ĐTB của cả 2 đội
 
 function champion() {
   const teamA = {
```

Comment:
- dòng 11, 21 viết hàm reduce lại, viết ntn chưa chặt chẽ. Viết lại
- Ý tưởng tốt, 1 like cho ý tưởng

Reply:
- Dòng 29 -> 34: Không viết được rõ ràng ra à. Chỉ sử dụng toán tử 3 ngôi cho 1 điều kiện thôi. Viết như thế này nó không được từng minh cho lắm

Reply:
Cách 2 thấy comment mà chưa thấy làm